fdbserver/include/fdbserver/LogSystem.h

```diff
@@ -229,12 +229,15 @@ struct ILogSystem {
 		int fastReplies;
 		int unknownReplies;
 
+		bool returnEmptyIfStopped; // valid only when unicast is enabled
```

Comment:
The source log server can return an empty version range only if this flag is set.